### PR TITLE
Fixes dog biscuit recipe not working as intended.

### LINKED ISSUE #8273: Dog biscuit recipe doesn't make dog biscuits!
### Labels

[MINOR]

### Description

The recipe for dog biscuits recipe doesn't make biscuits!

### Steps to reproduce

Get everything you need to make a dog biscuit (granola bar, meat paste and peanut butter), shove it into the oven for 6 seconds and TADA! You have....a hotdog!!!

### Expected Behavior

Biscuits...not hotdogs!

### Additional Information & Screenshots

Reported by: Shiiba
On server: Goonstation 4 Roleplay: Sylvester
Round log date: 2022-04-19-16-44
Reported on: 2022-04-19 17:51:47

### PR DESCRIPTION
**Pull request by Juggernaut2442**
[BUG - MINOR]

## About the PR 
Currently, the dog biscuit recipe outputs a hotdog, this is because the hotdog recipe (simple) is above the dog biscuit recipe (complex) in the code. This PR fixes that by putting the hotdog recipe under the "single-ingredient recipes" comment, since it only uses meatpaste.

## Why's this needed? 
Fixes #8273

### CODE REVIEW

**Review by ZeWaka — COMMENTED** [review]
hotdog

**Comment by Juggernaut2442** [comment]
hotdog

File: code/obj/submachine/cooking.dm [hunk 1]
```diff
@@ -530,7 +530,6 @@ table#cooktime a#start {
 			#endif
 			src.recipes += new /datum/cookingrecipe/cake_custom(src)
 			src.recipes += new /datum/cookingrecipe/meatloaf(src)
-			src.recipes += new /datum/cookingrecipe/hotdog(src)
```
Comment (frawhst) [inline code/obj/submachine/cooking.dm:533]: deleting hotdog........

File: code/obj/submachine/cooking.dm [hunk 2]
```diff
@@ -587,6 +586,7 @@ table#cooktime a#start {
 			src.recipes += new /datum/cookingrecipe/hardboiled(src)
 			src.recipes += new /datum/cookingrecipe/bakedpotato(src)
 			src.recipes += new /datum/cookingrecipe/rice_ball(src)
+			src.recipes += new /datum/cookingrecipe/hotdog(src)
```
Comment (frawhst) [inline code/obj/submachine/cooking.dm:589]: oh, we re add hotdog here. that's ok then

**Review by frawhst — APPROVED** [review]

**Pull request merged**